leetcode_python/Array/minimum-pair-removal-to-sort-array-ii.py

```
@@ -0,0 +1,123 @@
+        right[n - 1] = -1
+        alive = [True] * n
+        ver = [0] * n
+
+        bad = sum(1 for i in range(n - 1) if val[i] > val[i + 1])
+        heap = [(val[i] + val[i + 1], i, 0) for i in range(n - 1)]
+        heapq.heapify(heap)
+
+        ops = 0
+        while bad:
+            while True:
+                s, i, v = heapq.heappop(heap)
```

Ruff (0.16.1)

[warning] Unpacked variable `s` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)